packages/cli/src/PublicApi/v1/handlers/credentials/credentials.service.ts

```diff
@@ -87,8 +87,6 @@ export async function removeCredential(credentials: CredentialsEntity): Promise<
 }
 
 export async function encryptCredential(credential: CredentialsEntity): Promise<ICredentialsDb> {
```

Comment:
we should setup the `require-await` linting rule to get rid of unnecessary `async` markers like these.